### ISSUE TITLE
test_element_tree_import_script fail

### ISSUE THREAD

**Issue by MeggyCal** [sissaschool/xmlschema#167]
TestElementTree.test_element_tree_import_script fails with AssertionError: False is not true : Wrong import of ElementTree after xmlschema

This is similar to https://github.com/tiran/defusedxml/issues/28. HTH

**Comment by brunato** [comment]
Hi,
to try to fix this i need at least platform and Python version/implementation. Any further info about test run can be useful. The issue that you mention is the reason because I switched off *defusedxml* dependency and implemented a new safe XMLParser for ElementTree inside this library.

Thanks

**Comment by brunato** [comment]
Hi @MeggyCal,
any additional info about this issue?

**Comment by felixonmars** [comment]
I was getting the same error by running `python -m unittest` directly in the source directory without installing xmlschema. Setting `PYTHONPATH=$PWD` before running the tests fixes this for me. Hope this helps.

**Comment by brunato** [comment]
Hi,
this maybe due to missing package installation (*xmlschema* or the dependency *elementpath*). This test is based on an external script so it's conditioned by the package installation into the environment.

So I added an additional test to verify if the external script hasn't an import error. In this case an explicit message will be produced:

```
======================================================================
FAIL: test_element_tree_import_script (test_etree.TestElementTree)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/brunato/Development/projects/xmlschema/tests/test_etree.py", line 67, in test_element_tree_import_script
    "is missing:\n\n{}".format(stderr))
AssertionError: False is not true : Test script fails because a package is missing:

Traceback (most recent call last):
  File "/home/brunato/Development/projects/xmlschema/tests/check_etree_import.py", line 31, in <module>
    import xmlschema.etree
ModuleNotFoundError: No module named 'xmlschema'
```

**Comment by brunato** [comment]
More informative messages are available in case of test failure in release v1.1.2. The test is executed only on Linux platform because the external script needs *xmlschema*, and it might not available on some test setup of Windows or macOS platforms.
